pandas/io/parsers/readers.py

```diff
@@ -1287,13 +962,16 @@ def read_table(
     skipfooter: int = 0,
     nrows: int | None = None,
     # NA and Missing Data Handling
-    na_values: Sequence[str] | Mapping[str, Sequence[str]] | None = None,
+    na_values: Hashable
+    | Iterable[Hashable]
+    | Mapping[Hashable, Iterable[Hashable]]
+    | None = None,
     keep_default_na: bool = True,
     na_filter: bool = True,
     verbose: bool | lib.NoDefault = lib.no_default,
     skip_blank_lines: bool = True,
     # Datetime Handling
-    parse_dates: bool | Sequence[Hashable] = False,
+    parse_dates: bool | Sequence[Hashable] | None = None,
```

Comment:
Not sure what to do here: None of the three methods document `None` but it is the default in `read_csv`.

A few options:
- Document it, but keep it `False` in `read_table` (and add a deprecation warning, that it will be None in the future)
- Document it, change it to `None` in `read_table` (might need a deprecationwarnign first?)
- Disallow `None`, add a deprecation warning in the other methods

Reply:
> Document it, change it to None in read_table (might need a deprecationwarnign first? edit: could probably directly change it with 3.0 but add a whatsnew entry?)

I would support this and mark it as an `Other API changes`

Reply:
Okay, the current commits already do that :)